docs/settingUp.md

```diff
@@ -111,6 +113,26 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
+### IntelliJ IDEA
+
+1. Download and install [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/).
+   * You will need an Ultimate Edition license in order to work with Google App Engine in IntelliJ IDEA.
+   * You can sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
+1. Configure IntelliJ as follows:
+   * JRE: Click `Configure → Project Defaults → Project Structure`. Under `Project SDK`, click `New → JDK`. Locate the `Java` folder where you have installed `JDK 1.7`. Select `jdk1.7.*` and click `OK`.
+   * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentation. Go to `File → Settings → Editor → Code Style` and ensure that `Use tab character` is unchecked for `Java`, `JavaScript`, `HTML`, `CSS` and `XML`.
+   * Text Encoding: Go to `File → Settings → Editor → File Encodings` and ensure that `IDE Encoding` and `Project Encoding` are set to `UTF-8`.
+   * HTML/JSP syntax: We prefer not to use the HTML/JSP Inspections provided by IntelliJ. Go to `File → Settings → Editor → Inspections` and uncheck `HTML` and `JSP Inspections`.
+1. Import the project into IntelliJ.
+   * Click `Import existing project` and select the `teammates`* folder.
```

Comment:
Can consider just putting "...select the local repository folder." and removing line 134.